package/gluon-l3roamd/files/etc/modules.d/gluon-l3roamd

```diff
@@ -0,0 +1 @@
+cfg80211
```

Comment:
l3roamd soon will work without cfg80211 being loaded https://github.com/tcatm/l3roamd/pull/31
unfortunately it cannot know if cfg80211 is being loaded after it was started.
so either we should have l3roamd started by netifd or keep this hack... not sure which one is better...